lean-repo-utils.php

```diff
@@ -1,28 +1,28 @@
 <?php
 
-function load_github_secrets($gitHubSecretsFile)
+function load_git_secrets($gitSecretsFile)
 {
-  if (!file_exists($gitHubSecretsFile)) {
-    print "Could not find $gitHubSecretsFile\n";
+  if (!file_exists($gitSecretsFile)) {
+    print "Could not find $gitSecretsFile\n";
     return [];
   }
-  $gitHubSecretsContents = file_get_contents($gitHubSecretsFile);
-  if (empty($gitHubSecretsContents)) {
+  $gitSecretsContents = file_get_contents($gitSecretsFile);
+  if (empty($gitSecretsContents)) {
     print "GitHub secrets file is empty\n";
     return [];
   }
-  $gitHubSecrets = json_decode($gitHubSecretsContents, true);
-  if (empty($gitHubSecrets)) {
-    print "No data in GitHub secrets\n";
+  $gitSecrets = json_decode($gitSecretsContents, true);
+  if (empty($gitSecrets)) {
+    print "No data in Git secrets\n";
   }
-  return $gitHubSecrets;
+  return $gitSecrets;
 }
 
 /**
  * Read the secrets.json file
  */
 function pantheon_get_secrets($bindingDir, $requiredKeys, $defaultValues) {
-  $secretsFile = "$bindingDir/files/private/secrets.json";
+  $secretsFile = "$bindingDir/files/private/.build-secrets/tokens.json";
```

Comment:
Can we bump the major version of the pushback since we are making b/c breaks here? The debugging / diagnostic capabilities of the pushback are not great, and it would be a shame to break folk's workflow.

Reply:
Didn't realize this was part of this PR, tbh.
I'm not sure why we're changing this, so I'd rather keep it the same so we can stay on the same major version.

Reply:
@rvtraveller can you elaborate on why you changed this?

Current projects using version `1.x` don't have GitLab or BitBucket support. I'm not opposed to a major version bump and updating the `example-*` repos to use the new version for new projects.

If existing projects desire Quicksilver pushback for GitLab or BitBucket they can update to `2.x`. It might even be an unexpected change if it started working in `1.x`

Reply:
It was requested in https://github.com/pantheon-systems/terminus-build-tools-plugin/pull/193#pullrequestreview-223008934 to start using a directory for this instead of a file at the top level.

Reply:
@greg-1-anderson do you still feel strongly about a new directory for secret file storage and a major version bump? Perhaps we can continue without that change in this PR and make a follow-up PR with breaking changes.

Reply:
Yeah, it's okay to switch from secrets.json to some other file, but whether or not this different location is in a directory or at the root, if we do this it means b/c breaking changes to Quicksilver Pushback, which means we'd need a major version bump to avoid breaking existing users.

Since the Build Tools Plugin already switched to the new file, it seems that the best way forward is to continue with the major version bump here. However, if you prefer to roll back to secrets.json and avoid the major version bump. We just need to be consistent.

Reply:
> Since the Build Tools Plugin already switched to the new file, it seems that the best way forward is to continue with the major version bump here

I'm okay with a major version bump. 

> so I'd rather keep it the same so we can stay on the same major version.

@aaronbauman can you elaborate on why you do *not* want to do a major version bump? Are you concerned about existing projects?

Reply:
My reasoning is that this functionality doesn't actually _require_ breaking changes. Maybe we can create an analogous PR for both major versions?

Anyway, I'm not super concerned personally, as I'm currently running on this PR branch.